src/components/messaging/conversation.component.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import {
+  ImageProps,
+  View,
+  TouchableOpacity,
+  TouchableOpacityProps,
+} from 'react-native';
+import {
+  Conversation as ConversationModel,
+  Message,
+} from '@src/core/model';
+import {
+  ThemedComponentProps,
+  ThemeType,
+  withStyles,
+} from '@kitten/theme';
+import { DoneAllIcon } from '@src/assets/icons';
+import { Text } from '@src/components/common';
+import { ConversationInterlocutor } from './';
+
+interface ComponentProps {
+  index?: number;
+  conversation: ConversationModel;
+  onConversation: (index: number) => void;
+}
+
+export type ConversationProps = & ThemedComponentProps & TouchableOpacityProps & ComponentProps;
+
+class ConversationComponent extends React.Component<ConversationProps> {
+
+  private onConversation = (): void => {
+    this.props.onConversation(this.props.index);
+  };
+
+  private getLastMessageText = (): string => {
+    const { conversation } = this.props;
+    const lastMessage: string = conversation.messages[conversation.messages.length - 1].text;
+
+    return lastMessage.length <= 37 ? lastMessage : `${lastMessage.substring(0, 37)}...`;
```

Comment:
37?

Reply:
Just a temporary solution. We have to think about "text normalizing" to fit container width but I guess this is not the best time for it.